src/apps/vpn/connectionhealth.h

```diff
@@ -33,6 +33,12 @@ class ConnectionHealth final : public QObject {
 
   ConnectionStability stability() const { return m_stability; }
 
+  void overwriteStabilityForInspector(ConnectionStability stability) {
+    m_stabilityOverwritten = true;
```

Comment:
Nit: What do you think about checking that this cannot be called outside of dev mode ? :) 
```
if(Constants::inProduction()){
 qFatal << "Illegal invokation" 
}
```